oci/spec_opts_unix.go

```diff
@@ -357,8 +357,8 @@ func WithUIDGID(uid, gid uint32) SpecOpts {
 
 // WithUserID sets the correct UID and GID for the container based
 // on the image's /etc/passwd contents. If /etc/passwd does not exist,
-// or uid is not found in /etc/passwd, it sets gid to be the same with
-// uid, and not returns error.
+// or uid is not found in /etc/passwd, it sets the requested uid and gid
```

Comment:
I think the new wording can be misunderstood, as a direct reading is "sets the requested uid and gid to zero" meaning both will be set to zero. I think to be clearer it needs separate clauses: "sets the requested uid, additionally sets the gid to 0, and does not return an error."

Reply:
Bah, I reworded it several times to avoid exactly that but clearly ended up with a blind spot. I like your wording, will update.